File: common/src/main/java/org/tdf/common/store/ByteArrayMapStore.java
package org.tdf.common.store;

import org.tdf.common.util.ByteArrayMap;

import java.util.Map;

public class ByteArrayMapStore<V> extends MapStore<byte[], V> {
    public ByteArrayMapStore() {
        super(new ByteArrayMap<>());
    }


    public ByteArrayMapStore(Map<byte[], V> map) {
        super(map);
    }
}

File: common/src/main/java/org/tdf/common/store/ByteArrayMapStore.kt
package org.tdf.common.store

import org.tdf.common.util.ByteArrayMap

open class ByteArrayMapStore<V> : MapStore<ByteArray, V> {
    constructor() : super(ByteArrayMap<V>())
    constructor(map: MutableMap<ByteArray, V>) : super(map)
}